appendices/VK_KHR_sampler_ycbcr_conversion.txt

```diff
@@ -10,7 +10,6 @@ include::{generated}/meta/VK_KHR_sampler_ycbcr_conversion.txt[]
     No known IP claims.
 *Interactions and External Dependencies*::
   - Promoted to Vulkan 1.1 Core
-  - This extension interacts with `<<VK_EXT_debug_report>>`
```

Comment:
This seems like a copy-paste error.
@fluppeteer Did you have any other extension in mind here?